### PR TITLE
add conda env for class 2d ranker

### PR DESCRIPTION
**Pull request by azazellochg**
Removed RELION_PYTHON, replaced by conda env variable RELION_ENV_ACTIVATION

### CODE REVIEW

File: relion/__init__.py [hunk 1]
```diff
@@ -93,6 +121,10 @@ def defineBinaries(cls, env):
                           (f'make -j {env.getProcessors()}',
                            ['bin/relion_refine'])]
 
+            if ver != V3_1:
+                installCmd.append(('%s conda create -y -n relion-python python=3 numpy pytorch' %
```
Comment (delarosatrevin) [inline relion/__init__.py:125]: There is no package installed in the newly created environment?

**Comment by azazellochg** [comment]
Nope, no packages. Just numpy and torch.

**Comment by delarosatrevin** [comment]
relion-python is shown in the list, but I get an error when trying to install it

```
josem@artemis:~/work/development/scipion/scipion3/source/plugins/scipion-em-relion$ scipion installb --help
...
relion                   3.1     [ ]     4.0     [X]     
relion-python            4.0     [ ]     
resmap                   1.95    [ ]     
spider                   26.06   [ ]     
topaz                    0.2.3   [ ]     0.2.4   [X]     
xmippSrc                 3.22.04.0[ ]     
josem@artemis:~/work/development/scipion/scipion3/source/plugins/scipion-em-relion$ scipion installb relion-python
Scipion v3.0.10 - Eugenius
ERROR: Could not find target relion-python
```

**Review by delarosatrevin — APPROVED** [review]
Now working!

**Pull request merged**